core/vm/contract.go

```diff
@@ -84,10 +81,44 @@ func NewContract(caller ContractRef, object ContractRef, value *big.Int, gas uin
 	return c
 }
 
+func (c *Contract) validJumpdest(dest *big.Int) bool {
+	udest := dest.Uint64()
+	// PC cannot go beyond len(code) and certainly can't be bigger than 63bits.
+	// Don't bother checking for JUMPDEST in that case.
+	if dest.BitLen() >= 63 || udest >= uint64(len(c.Code)) {
+		return false
+	}
+	// Only JUMPDESTs allowed for destinations
+	if OpCode(c.Code[udest]) != JUMPDEST {
+		return false
+	}
+	var analysis bitvec
```

Comment:
You don't use this anywhere outside of the nested if below. You could delete it from here and instead of declaring `exists` explicitly below too, just do `analysis, exist := c.jumpdests[c.CodeHash]`.

Did you want to use this for something and perhaps forgot?

Reply:
You're right, it can be simplified. I iterated a bit on that loop, and it's probably some leftover from that